config/v1alpha1/types_cluster_monitoring.go

```diff
@@ -77,6 +79,13 @@ type ClusterMonitoringSpec struct {
 	// userDefined set the deployment mode for user-defined monitoring in addition to the default platform monitoring.
 	// +required
 	UserDefined UserDefinedMonitoring `json:"userDefined"`
+
+	// metricsServer defines the configuration for the Metrics Server component.
```

Comment:
What does this field allow a user to configure about the metrics server component? (i.e why would I, as a user, care to do anything with this field?)